<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="xh">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>20</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/picture/xh.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>沈航校园导航系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pu_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>主界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pu_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>景点查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pu_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导航界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pu_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>电子地图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pu_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cx_edit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>631</width>
      <height>301</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="znj">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/znz.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="poem2">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>行车不规范，亲人两行泪</string>
    </property>
   </widget>
   <widget class="QLabel" name="poem1">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>道路千万条，安全第一条</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>790</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>于2019年12月18日制 作者：软件1801何雨泊</string>
    </property>
   </widget>
   <widget class="QLabel" name="cx_label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>611</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>欢迎您的查询，请选择您要查询的景点</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cx">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>行政楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经管楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>逸夫馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>工程训练中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>航空宇航馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合教学楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机械馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>艺术馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>体育馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合体育场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生活动中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蓝天剧场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生飞行器创新实践基地</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>创新创业楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>国际教育学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区足球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>友谊会馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区篮球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>校医院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区运动场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通航实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重点学科实验室</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>热能实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蒲新教师公寓</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="main_pic">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>1021</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/qyh.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cx_begin_na">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>请选择导航起点：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cx_end_na">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>请选择导航终点：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dh_begin">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>行政楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经管楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>逸夫馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>工程训练中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>航空宇航馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合教学楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机械馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>艺术馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>体育馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合体育场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生活动中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蓝天剧场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生飞行器创新实践基地</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>创新创业楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>国际教育学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区足球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>友谊会馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区篮球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>校医院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区运动场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通航实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重点学科实验室</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>热能实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蒲新教师公寓</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dh_end">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>行政楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>经管楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>逸夫馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>工程训练中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>航空宇航馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合教学楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>图书馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>机械馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>艺术馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>体育馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>综合体育场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生活动中心</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蓝天剧场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大学生飞行器创新实践基地</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>创新创业楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>国际教育学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区足球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>友谊会馆</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区篮球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>校医院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>南区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>网球场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区食堂</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区学生公寓</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北区运动场</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通航实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重点学科实验室</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>热能实验楼</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>蒲新教师公寓</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="all_line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="min_line">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="all_na">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>两地点的所有路径：</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_na">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>两地点的最短路径：</string>
    </property>
   </widget>
   <widget class="QLabel" name="map">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>811</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/picture/map.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
